R/wb_add_flextable.R

```diff
@@ -40,6 +40,11 @@ wb_add_flextable <- function(wb, sheet, ft,
                              dims = NULL) {
   # Check input
   stopifnot("wbWorkbook" %in% class(wb))
+
+  if (inherits(sheet, "openxlsx2_waiver")) {
+    sheet <- wb$validate_sheet(sheet)
+  }
+
```

Comment:
using `sheet <- wb$validate_sheet(sheet)` should be enough. The `stopifnot()` below is probably unneeded.